app/views/exports/pbmc_bank_tubes_content_report.csv.erb

```diff
@@ -0,0 +1,83 @@
+<%# This is a report of the contents of the tubes in the PBMC Bank plate %>
+<%# Based on app/views/exports/hamilton_lrc_pbmc_bank_to_lrc_bank_seq_and_spare.csv.erb %>
+<%= CSV.generate_line ['Workflow', @workflow], row_sep: '' %>
+
+<%=
+  CSV.generate_line [
+                      'Well name',
+                      'Donor ID',
+                      'Parent barcode',
```

Comment:
I know this is what it says in the story (my fault, not sure where it came from in the first place), but could we call this "Stock barcode" instead? I think "Parent barcode" is misleading because the LRC Blood Vac tube is not the parent labware of the LRC Bank Seq / Spare tubes. Thanks.